mail/api_test.py

```diff
@@ -77,8 +77,10 @@ def test_messages_for_recipients():
         assert msg.subject == "Welcome {}".format(user.profile.name)
 
 
-def test_send_message(mailoutbox):
+def test_send_message(mocker, mailoutbox):
     """Tests that send_messages works as expected"""
+    sendmail = mocker.patch('mail.api.AnymailMessage.send', side_effect=ConnectionError)
```

Comment:
Could you do this as a separate failure test case? this is supposed to be a affirmative test case